tools/report.sh

```diff
@@ -283,16 +282,17 @@ if [[ -n $DC_DEPLOY ]]; then
 fi
 
 echo "customresources"
-mkdir -p "$OUT_DIR"/reports/crds "$OUT_DIR"/reports/crs
-CRDS=$($KUBE_CLIENT get crd -l app=strimzi -o name | cut -d "/" -f 2) && readonly CRDS
+mkdir -p "$OUT_DIR"/reports/customresourcedefinitions
```

Comment:
It says custom resources above. So why is the directory suddenly named custom resource definitions?

Reply:
This code block gets both CRDs and CRs, I didn't change that. It's similar to the cluster operator block that gets different resource types.